openedx/core/djangoapps/site_configuration/models.py

```diff
@@ -32,6 +33,11 @@ class SiteConfiguration(models.Model):
         blank=True,
         load_kwargs={'object_pairs_hook': collections.OrderedDict}
     )
+    receipt_page_url = models.CharField(
+        max_length=255,
+        default='/commerce/checkout/receipt/?orderNum=',
```

Comment:
This default value is a duplicate of the failover value used [here](https://github.com/edx/edx-platform/pull/13299/files#diff-6bfaaa3703d73979b336ebbd7a656181R67) -- it probably makes sense to move it into a constant or configuration setting